### PR TITLE
feat: wrapped/typed translator function

### PR DESCRIPTION
**Pull request by lostfictions**
#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13663/improve-localisation-library

based on https://github.com/PeerioTechnologies/peerio-translator/pull/2 

and pending a desktop pr to switch all uses of `t` to the one exported here

#### Testing instructions  

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -141,5 +138,13 @@
...
     "npm": "5.x"
   },
-  "nyc": {}
+  "nyc": {},
+  "husky": {
+    "hooks": {
+      "commit-msg": "validate-commit-msg",
+      "post-checkout": "npm run opt-in-npm-install",
+      "post-merge": "npm run opt-in-npm-install",
+      "pre-commit": "makedefs src/copy/en.json src/copy/defs.d.ts && git stage src/copy/defs.d.ts && opt --in lint --exec \"npm run validate\""
```
Comment (lostfictions) [inline package.json]: feels a bit icky/clunky but i guess it should be ok?

Comment (anri-asaturov) [reply package.json]: yeah, we can make a separate shell script if it gets any longer

**Comment by anri-asaturov** [comment]
Excuse me, I'm a bit lost, so we have a translator PR, now this PR, and I'm waiting for desktop and mobile PR to merge all of them together?

**Comment by lostfictions** [comment]
@anri-asaturov i don't know what the mobile situation is.

i'm just verifying that there's nothing missing here that's needed in desktop, and then the translator pr can be merged, and this can be merged. the desktop pr will just point all files to use `t` from icebear instead of from the translator package directly.

**Comment by anri-asaturov** [comment]
Hmm, right, mobile is not typechecking and api didn't change so it's fine

**Comment by lostfictions** [comment]
should be ok to merge independently of desktop pr

**Pull request merged**